packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -28,13 +28,20 @@ import { disposeAllLocalPreviewViews, _isInCall, _isPreviewOn } from '../utils/c
 
 /* @conditional-compile-remove(PSTN-calls) */
 /**
- * handler type for addParticipant
- *
+ * Handler type for addParticipant
  * @beta
  */
 export type AddParticipantHandler = ((participant: CommunicationUserIdentifier) => Promise<void>) &
   ((participant: PhoneNumberIdentifier, options: AddPhoneNumberOptions) => Promise<void>);
 
+/* @conditional-compile-remove(PSTN-calls) */
+/**
+ * Handler for removeing a participant from a call.
+ * @beta
+ */
+export type RemoveParticipantHandler = ((userId: string) => Promise<void>) &
```

Comment:
out of interest is there a reason to not do:
```
(userId: string | CommunicationIdentifier) => Promise<void>
```
here?

Reply:
Looking above this I guess the `&` style matches the `AddParticipantHandler`

Reply:
This format will help us when we need to deprecate one of the overloads
(userId: string) in the future.
<img width="764" alt="Screen Shot 2022-09-30 at 1 31 01 PM" src="https://user-images.githubusercontent.com/9782747/193351676-793efc67-f25b-463d-9471-a14cfec70da6.png">
<img width="684" alt="Screen Shot 2022-09-30 at 1 31 18 PM" src="https://user-images.githubusercontent.com/9782747/193351705-fc5e3c4f-4103-4712-b1fe-9043189b9538.png">

Reply:
For deprecated the (userId: string) parameter, it's kinda like we are removing one generic api but couple the adpter entirely to our ACS? I would say adding a new parameter sounds good to me, but deprecating the current one still need some discussions (and I don't think we are going to do this in this PR)

Reply:
Noticed this is for handler not adapter - still handler apis are designed for component and meant to be not bound with ACS, while adding ACS type is good, we might still want to remain a generic way to implement this